bitnami/spring-cloud-dataflow/values.schema.json

```diff
@@ -324,6 +324,12 @@
           "title": "Use a new RabbitMQ server hosted in the cluster",
           "form": true,
           "description": "Whether to deploy a RabbitMQ server to satisfy the Spring Cloud Skipper messaging middleware requirements."
+        },
+        "service" : {
+          "port": {
+            "type": "integer",
+            "title": "AMQP port presented in SPRING_RABBITMQ_PORT"
+          }
```

Comment:
These changes should also be discarded, as we are no longer adding a new property to the `values.yaml` file.